app/views/courses/delete.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= content_for :page_title, "Delete" %>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl"><%= @course.name %> (<%= @course.course_code %>)</span>
+        Are you sure you want to delete this course?
+      </h1>
+
+      <p class="govuk-body">You can only delete a course if it hasn’t been published in this cycle.</p>
+
+      <p class="govuk-body">Delete a course if you:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>created it by mistake</li>
+        <li>won’t offer it again</li>
+      </ul>
+
+      <p class="govuk-body">Deleting a course is permanent – you can’t undo it. If you need to recreate a course after deleting it, it will have a new course code.</p>
+
+      <h2 class="govuk-heading-m">Contact support to delete this course</h2>
+
+      <p class="govuk-body">Contact the <%= mail_to 'becomingateacher@digital.education.gov.uk', 'Becoming a teacher team', class: "govuk-link" %> by email to delete this course.</p>
```

Comment:
Can we put the course code and name into the email subject automatically?

Reply:
same with withdraw i guess